### PR TITLE
Fix Hamburger menu in RTL

### PR DESCRIPTION
**Pull request by ahmed-salaah**

### CODE REVIEW

File: shellXamarin.iOS/Info.plist [hunk 1]
```diff
@@ -36,9 +36,9 @@
 	<string>Assets.xcassets/AppIcon.appiconset</string>
 	<key>CFBundleLocalizations</key>
 	<array>
```
Comment (ahmed-salaah) [inline shellXamarin.iOS/Info.plist]: It fixes the alignment when  make it only ar

**Comment by mostafasany** [comment]
👍

File: Module/shellXamarin.Module.Common/Services/ILayoutDIrectionService.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+namespace shellXamarin.Module.Common.Services
```
Comment (mostafasany) [inline Module/shellXamarin.Module.Common/Services/ILayoutDIrectionService.cs]: file name still with I capital

File: shellXamarin.iOS/shellXamarin.iOS.csproj [hunk 3]
```diff
@@ -159,11 +162,16 @@
       <Project>{42D99B98-7E17-429D-AD52-C91503647293}</Project>
       <Name>shellXamarin</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Module\shellXamarin.Module.Common\shellXamarin.Module.Common.csproj">
```
Comment (mostafasany) [inline shellXamarin.iOS/shellXamarin.iOS.csproj]: do we need to add common project to IOS?

File: shellXamarin.iOS/Info.plist [hunk 4]
```diff
@@ -36,9 +36,9 @@
...
-		<string>en</string>
 		<string>ar</string>
-		<string>de</string>
 	</array>
+	<key>CFBundleDevelopmentRegion</key>
```
Comment (mostafasany) [inline shellXamarin.iOS/Info.plist]: Is this important ?

File: shellXamarin.iOS/shellXamarin.iOS.csproj (hunk 3 above)
Comment (ahmed-salaah) [reply shellXamarin.iOS/shellXamarin.iOS.csproj]: yes because of the ILayoutDirection

**Pull request merged**